docker/standup_base

```diff
@@ -26,6 +26,4 @@ COPY requirements.txt /app/requirements.txt
 RUN pip3 install --require-hashes --no-cache-dir -r requirements.txt
 
 COPY . /app
-
-RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
-RUN chown webdev.webdev -R .
+RUN DEBUG=False SECRET_KEY=foo DATABASE_URL=sqlite:// ./manage.py collectstatic --noinput
```

Comment:
I wonder if there's a better way to run collectstatic other than setting these things in the environment. Are there other management commands we'd want to run ignoring these variables, too? Should we make a bash wrapper for it? Something like `phb_manage.sh`:

```
#!/bin/bash

# phb_manage.sh runs manage.py with oblivious hand-wavey default settings
# suitable for management commands that will work despite the wildly ignorant
# nature of hand-wavey default settings.
#
# Usage: phb_manage.sh [COMMAND] [ARGS]
#
# Example: phb_manage.sh collectstatic --noinput

DEBUG=False
SECRET_KEY=foo
DATABASE_URL=sqlite://

./manage.py $@
```

Then we can use it elsewhere and it's probably easier to remember to update maybe? I'm just thinking out loud.